builtin/commit-graph.c

```diff
@@ -55,9 +56,11 @@ static int graph_verify(int argc, const char **argv)
 			   N_("The object directory to store the graph")),
 		OPT_BOOL(0, "shallow", &opts.shallow,
 			 N_("if the commit-graph is split, only verify the tip file")),
+		OPT_BOOL(0, "progress", &opts.progress, N_("force progress reporting")),
 		OPT_END(),
 	};
 
+	opts.progress = 1;
```

Comment:
@garimasi514 I noticed that several of the other verbs (e.g. `pack-objects`, `fetch`, `merge`) will also check `isatty(2)` when determining if git should show progress (with `--progress` forcing progress to be written even if `isatty(2)` is `1`).  Do we need to do something similar here?

I also noticed that some verbs use `-q` or `--quiet` to prevent progress from being written rather than `--no-progress`, do you know if there are any guidelines for which option we use?

cc: @derrickstolee @jeffhostetler for their thoughts as well

Reply:
> with --progress forcing progress to be written even if isatty(2) is 1

From what I could tell this is why the description of the `--progress` option is `"force progress reporting"` for the other verbs.

Reply:
@garimasi514 ok thanks!  I will do this for the MIDX changes as well.